installation/resources/installer-config-local.yaml.tpl

```diff
@@ -39,6 +39,7 @@ data:
   global.minikubeIP: ""
   global.ingress.domainName: "kyma.local"
   global.externalServicesMock.enabled: "true"
+  global.externalServicesMock.auditlog: "true"
```

Comment:
It's probably not a problem, but I just noticed this:
1. We set auditlog: true here
2. We also set it in two places in the test-infra scripts:
- https://github.com/kyma-project/test-infra/blob/a606c71b5a88cd86e1d05263427488cd51a8f826/prow/scripts/cluster-integration/compass-gke-integration.sh#L236
- https://github.com/kyma-project/test-infra/blob/e3fb15e8805853a02037df4bb57b78bf0f0c31d4/prow/scripts/cluster-integration/compass-gke-benchmark.sh#L237

It's probably not an issue that it's set twice, but I just wanted to point it out, because you're more in the context so you might know something more if it's an issue. (if it's not an issue, we might even go ahead and remove it from test-infra?)

Reply:
The configuration from test infra is for the tests deployments and it is needed in order to start the auditlog component of the external-services-mock. This configuration here is for local installations